modules/ocf_desktop/manifests/init.pp

```diff
@@ -7,7 +7,9 @@
   include ocf::packages::pulse
   include ocf::packages::vscode
 
-  include ocf_mesos::slave
```

Comment:
Are we still intending to use desktops as media slaves at all, given that we're moving to Kubernetes?

Reply:
I don't plan to do this for Kubernetes.

Reply:
*mesos

Reply:
Might as well completely get rid of this code then

Reply:
It's completely gone as of #680 anyway